torch/nn/functional.py

```diff
@@ -3839,27 +3839,133 @@ def fold(input, output_size, kernel_size, dilation=1, padding=0, stride=1):
 
 def _pad_circular(input, padding):
     # type: (Tensor, List[int]) -> Tensor
+    """Circularly pads tensor.
+
+    Tensor values at the beginning are used to pad the end, and values at the
+    end are used to pad the beginning. The first and second axes of the tensor
+    are not padded.
+
+    Args:
+        input: Tensor with shape :math:`(N, C, D[, H, W])`.
+        padding: Tuple containing the number of elements to pad each side of
+            the tensor. The length of padding must be twice the number of
+            paddable axes. For example, the length of padding should be 4 for a
+            tensor of shape :math:`(N, C, H, W)`, and the length should be 6
+            for a tensor of shape :math:`(N, C, D, H, W)`.
+
+    Examples::
+
+        >>> x = torch.arange(6).view(1, 1, 2, 3)  # Create tensor
+        >>> # Example 1
+        >>> padding = (1, 1, 1, 1)
+        >>> y = F.pad(x, padding, mode='circular')
+        >>> print(x)
```

Comment:
the original tensor is unmodified, right? so I don't think you need to show x

Reply:
Done.